### ISSUE TITLE
[Training] Custom weight init for RNNs

### ISSUE THREAD

**Issue by kapiblue** [JankowskiDaniel/Neural-Deep-Retina#102]
Since we have observed vanishing gradients (especially when training with the final Sigmoid activation) let's consider custom weight initialization for the CfC/LSTM components.

Could be useful:
[Tricks for training RNNs](https://www.scs-europe.net/conf/ecms2015/invited/Contribution_Zimmermann_Grothmann_Tietz.pdf)

**Comment by kapiblue** [comment]
We used [Xavier init](https://saturncloud.io/blog/how-to-initialize-weights-in-pytorch-a-guide-for-data-scientists/#3-xavier-initialization) in the bachelor's thesis and it is suitable for Sigmoid/tanh activations, I don't know precisely if it could be used with RNNs.

**Comment by JankowskiDaniel** [comment]
Xavier initialization is already implemented in the CfC under the hood
